pom.xml

```diff
@@ -78,6 +78,7 @@
 
     <mavenVersion>3.5.0</mavenVersion>
     <atlasVersion>0.17.1</atlasVersion>
+    <daVersion>2.1.0.Api-1</daVersion>
```

Comment:
Wow, what a GAV!! :-)

Reply:
Yeah, we needed to make a partial release because of a snapshot transitive dependency we cannot release ATM, so to emphisize that we chose this version suffix.

Reply:
Ok!